HACKING.md

```diff
@@ -17,6 +17,26 @@ On Debian/Ubuntu:
 
     $ sudo apt install nodejs
 
+
+Creating VM images locally (not necessary for running tests) needs the
```

Comment:
This is true, but it's quite out of place here. We have a separate repository which contains the image creation scripts https://github.com/cockpit-project/bots/

I would prefer the instruction on how to use these scripts  out of this document. Additionally building images locally is something we do seldom, as we have github actions that do it for us on a weekly basis.

Reply:
I see, I added it because there's the command for installing virt-install for fedora in the document. Do we plan to remove that as well or keep it?